gooddata-sdk/gooddata_sdk/catalog/base.py

```diff
@@ -65,3 +70,7 @@ def client_class() -> Any:
     def to_api(self) -> Any:
         dictionary = self._get_snake_dict()
```

Comment:
How does attrs map `from_` to `_from`? If it works out of box, please document it somewhere.

Reply:
There is no endpoint to edit the dependent entities graph, so the method `to_api` is never called. For that, reason I did not modify `to_api`. When the `to_api` method will be called on the dependent entities graph  it will raise an exception in the current setup. I propose to override this method for the dependent entities graph and return `NotImplemented`.